src/book/01-team/03-history/03-publications.mdx

```diff
@@ -2,6 +2,162 @@
 section: The NUbots Team
 chapter: History
 title: Publications
-description: 'A comprehensive list of NUbots academic publications.'
+description: Discover all of the NUbots publications.
 slug: /team/publications
 ---
+
+## 2002
+
+- [Chalup, S.K., Creek, N., Freeston, L., Lovell, N.G., Marshall, J., Middleton, R., Murch, C.L., Quinlan, M., Shanks, G., Stanton, C., & Williams, M. (2002). When NUbots Attack ! The 2002 NUbots Team Report.](https://pdfs.semanticscholar.org/d6eb/35e3b0b64483b27f20b752726b13916a64cd.pdf?_ga=2.204012112.173948975.1563246156-1496945461.1563246156)
+
+## 2003
+
+- [Chalup, Stephan & Coleman, Oliver & N. Freeston, Michaela & Middleton, R.H. & L. Murch, Craig & J. Quinlan, Michael & J. Seysener, Christopher & D. Shanks, Graham. (2003). The NUbots' Team Description for 2003. ](https://pdfs.semanticscholar.org/ae43/a1f8cb40da2f4b9823bc566d330515944dcc.pdf)
+
+- [Quinlan M.J., Murch C.L., Middleton R.H., Chalup S.K. (2003) Traction Monitoring for Collision Detection with Legged Robots. In: Polani D., Browning B., Bonarini A., Yoshida K. (eds) RoboCup 2003: Robot Soccer World Cup VII. RoboCup 2003. Lecture Notes in Computer Science, vol 3020. Springer, Berlin, Heidelberg.](https://doi.org/10.1007/978-3-540-25940-4_33)
+
+- [Bunting, Jared & Chalup, Stephan & Freeston, Michaela & McMahan, Will & Middleton, R.H. & Murch, Craig & Quinlan, Michael & Seysener, Christopher & Shanks, Graham. (2003). Return of the NUbots! The 2003 NUbots Team Report.](https://www.researchgate.net/profile/RH_Middleton/publication/247406435_Return_of_the_NUbots_The_2003_NUbots_Team_Report/links/00b7d52d3765f8a76c000000.pdf)
+
+- [Quinlan, M.J., Chalup, S.K., & Middleton, R.H. (2003). Techniques for Improving Vision and Locomotion on the Sony AIBO Robot.](https://s3.amazonaws.com/academia.edu.documents/42378432/QuinlanEtAlACRA2003.pdf?response-content-disposition=inline%3B%20filename%3DTechniques_for_Improving_Vision_and_Loco.pdf&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWOWYYGZ2Y53UL3A%2F20190716%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190716T031000Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=d8ab79bce2e84083cd3c81a1a7eaccd29237707df044696bbd8d217ebb33589b)
```

Comment:
I think the link here was temporary and has expired. Perhaps just link to the page on academia.edu.

Reply:
Ah yeah thanks for catching that. I'll go through them when I have time and try to find doi links where possible too. And download the papers and add them.